data/tokens.json

```diff
@@ -1251,32 +1251,28 @@
       "value": "{gtmds_size-xl}*2+{gtmds_size-sm}*3",
       "type": "sizing"
     },
-    "gtmds_action-item_indent-base": {
-      "value": "{gtmds_size-sm}+{gtmds_size-lg}",
-      "type": "spacing"
-    },
-    "gtmds_action-item_indent-0": {
+    "gtmds_action-item_padding-left-0": {
       "value": "{gtmds_size-md}",
       "type": "spacing"
     },
-    "gtmds_action-item_indent-1": {
-      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}",
+    "gtmds_action-item_padding-left-1": {
+      "value": "{gtmds_action-item_item_padding-left-0}+({gtmds_size-sm}+{gtmds_size-lg}",
       "type": "spacing"
     },
-    "gtmds_action-item_indent-2": {
-      "value": "{gtmds_action-item_indent-0}+{gtmds_action-item_indent-base}*2",
+    "gtmds_action-item_item_padding-left-2": {
```

Comment:
you have ...action-item_item_...
is the item twice intentional?

Reply:
Pushed new commit